communities/UNAM-Engineering/community-assessment.md

```diff
@@ -0,0 +1,162 @@
+
+We're open to projects that are related to CS or CE so we might talk about other topics and how people 
+solved them with code.
+
+### Goals
+- Why do people get involved in your community
+
+Most of the people that join is to share their code with the organization or to ask for code exmaples about
+their classes.
+
+- Are they students hoping to get a degree
+
+Yes, most of them. We also have graduates as part of the community.
+
+- Are they trying to ramp-up to get a job
+
+Most are looking for internships for summer/winter.
+
+- Are they passionate about developing, regardless of their major
+
+Yes, we have also psychologist and law students that love to code.
+
+- What activities and goals can you set for your community based on their existing goals
+
+  - Sharing Code.
+  - Learning from other students.
+  - Students meeting each other.
+  - Building a public "Portfolio" of the projects that students are making in the univesity.
+
+
+### Location
+- Where does your community currently gather?
+
+We talk via FB Messenger and Github. 
+
+- Is it an online-only group so far, or is there a place where people can meet?
+
+It is online so far though we could meet at the lab I'm working in.
+
+- Is there parking, and if so, is it free?
+
+There's public parking close though it's highly saturated.
+
+- Is there public transportation available to and from meet-ups?
+
+Yes, we are connectd to the two main public transportation systems in Mexico.
+
+- Is the location accessible to those both on and off campus?
+
+It's accesible for both.
+
+- How much does it cost to use the space?
+
+For students the spaces are free you just need to ask for it which sometimes can be a long process.
+
+- How flexible is the space for different types of presentations and meetings?
+
+For meetups and presentations in general is preaty flexible. for hackathons is more complicated and, so far,
+we haven't been able to organize one on campus.
+
+- How many people can it fit?
+
+Depends on the space you ask for. It goes from 40-160 and in special occations you can get a 440 space. 
+
+- Is there comfortable seating for all shapes and sizes?
+
+Yes
+
+- Is there water or snacks available nearby?
+
+Yes
+
+
+### Communication
+- How does your community get in touch with one another?
+
+We use FB Messenger and in rare ocations GH issues. 
+
+- Is there an email group, a GitHub organization, a Twitter account, a Facebook group, a chat room or channel?
+
+We have the Github organization as the main contact form.
+
+- Could your organization benefit from one or more of those, or another way of communicating?
+
+Yes we could. We could explore better ways to connect al members.
+
+### Culture
+- How welcoming is your community to new people and new ideas?
+
+We're open to new people and ideas.
+
+- What happens if there is a disagreement?
+
+We usually don't have disagreements. In the worst case escenario we vote for the best solution and include memebers.
```

Comment:
As the community grows beyond the core group of friends, this becomes harder to manage and a formal process can help. Definitely consider a code of conduct if you're looking to grow!